ts/screens/wallet/payment/ScanQrCodeScreen.tsx

```diff
@@ -184,8 +185,17 @@ class ScanQrCodeScreen extends React.Component<Props, State> {
       const maybePosteDataMatrix = decodePosteDataMatrix(data);
 
       return maybePosteDataMatrix.foldL<void>(
-        this.onInvalidQrCode,
-        this.onValidQrCode
+        () => {
+          if (this.state.scanningState !== "INVALID") {
+            void mixpanelTrack("WALLET_SCAN_POSTE_DATAMATRIX_FAILURE");
```

Comment:
Could you add to https://docs.google.com/spreadsheets/d/1bXNRcTDKU-5l2RqCcwSbT7Y1M9QQTWtS_f-3-AIqSYk/edit#gid=0 these new two events?

Reply:
@fabriziofff can you give me writing permissions on that file?

Reply:
Ops, you should have all the permissions now!

Reply:
@fabriziofff Done ✅